blocked_domains.json

```diff
@@ -5,6 +5,11 @@
             "domains": ["activitypub-troll.cf", "misskey-forkbomb.cf", "repl.co"],
             "severity": "suspend",
             "reason": "Spam"
+        },
+        {
+            "domains": ["threads.net", "threads.instagram.com"],
+            "severity": "suspend",
+            "reason": "https://ploum.net/2023-06-23-how-to-kill-decentralised-networks.html"
```

Comment:
Maybe a more concise "EEE" or "Embrace, Extend, Extinguish" would be more fitting, but I don't really have a preference. Just food for thought.